.gitignore

```diff
@@ -36,12 +36,12 @@ venv
 build
 deepinv.egg-info
 dist
-datasets
+examples/datasets
 wandb/
 kernels
 measurements/
 degradations/
-datasets/
+examples/*/datasets
```

Comment:
same question here?

Reply:
same as above, 
I inferred that the expected behavior was : 
ignoring all "datasets" subfolders in "examples" folder